.circleci/config.yml

```diff
@@ -230,6 +230,18 @@ jobs:
     environment:
       BROWSER: << parameters.browser >>
       BVER: << parameters.bver >>
+    trigger-qe-tests:
```

Comment:
@DenissSazanovs, Looks like this indentation for this block is off causing it to fail the build (https://circleci.com/gh/twilio/twilio-video.js/10107?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)

Please fix the indentation, and verify that the build passes.

Reply:
Fixed